src/lib.rs

```diff
@@ -283,17 +290,17 @@ pub fn parse_path(path: &PathBuf, config: ParseConfig) -> anyhow::Result<ParseOu
                     .failures
                     .push((id.clone(), format!("{failure_reason}")));
             }
+            let mut cid = e.compile_id.clone();
+            if let Some(c) = cid.as_mut() {
+                c.attempt = 0;
```

Comment:
Why do you need to set all the attempts to 0 in the compile id here?

Reply:
There's a modeling problem in the trace format.

The stack trace only ever gets emitted on attempt 0; we don't emit stack traces for retries for other attempts. But for the stack trie, obviously we are putting in every attempt. So I need to remap them to zero. A helper function would be a good idea, as would refactoring CompileId to have a non-attempt inner struct, and an attempt outer struct.